client/state/lib/middleware.js

```diff
@@ -227,6 +230,15 @@ const handler = ( dispatch, action, getState ) => {
 				if ( desktopEnabled ) {
 					updateSelectedSiteForDesktop( dispatch, action, getState );
 				}
+
+				const state = getState();
```

Comment:
Can we extract this code to a separate function? For example, `updateSelectedSiteForDesktop` a few lines above is just a one-line call, too.